src/providers/microsoft/azure/mod.rs

```diff
@@ -403,7 +403,7 @@ impl MetadataProvider for Azure {
         let certs_endpoint = match goalstate.certs_endpoint() {
             Some(ep) => ep,
             None => {
-                warn!("SSH pubkeys requested, but not provisioned for this instance");
+                warn!("SSH pubkeys not provisioned for this instance");
```

Comment:
I think the "requested" wording in this warning refers to the fact that *something* called `afterburn --ssh-keys USER` but we couldn't find anything. And so at this layer of this stack, the warning as written previously was accurate. (Even though we know that at the FCOS layer, it's because we unconditionally run this service for the `core` user.)

Reply:
what does `afterburn --ssh-keys USER` imply, though?

It's hard not to look at this from the lens of FCOS where we are saying essentially "if there are SSH keys give them to me".

Reply:
After looking at the code for the other providers and the history of this line, I think we should drop this warning entirely. Other providers don't warn in the case that no SSH keys were found, and AFAICT there is no reason to treat Azure differently.